noxfile.py

```diff
@@ -1,105 +1,134 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-import shutil
+from os             import getenv
 from pathlib        import Path
 
 import nox
-from setuptools_scm import ScmVersion, get_version
+from nox.sessions   import Session
 
 ROOT_DIR  = Path(__file__).parent
 
-BUILD_DIR = (ROOT_DIR  / 'build')
-CNTRB_DIR = (ROOT_DIR  / 'contrib')
-DOCS_DIR  = (ROOT_DIR  / 'docs')
-DIST_DIR  = (BUILD_DIR / 'dist')
+BUILD_DIR = ROOT_DIR  / 'build'
+CNTRB_DIR = ROOT_DIR  / 'contrib'
+DOCS_DIR  = ROOT_DIR  / 'docs'
+DIST_DIR  = BUILD_DIR / 'dist'
+
+IN_CI           = getenv('GITHUB_WORKSPACE') is not None
+ENABLE_COVERAGE = IN_CI or getenv('TORII_TEST_COVERAGE') is not None
 
 # Default sessions to run
 nox.options.sessions = (
 	'test',
 	'lint',
-	'typecheck'
+	'typecheck-mypy'
```

Comment:
Should this also include `'typecheck-pyright'`?

Reply:
Not for a default run, I don't think so,

Reply:
Okie, fair enough